SemanticUI/Modules/HtmlBuilder.elm

```diff
@@ -0,0 +1,49 @@
+module SemanticUI.Modules.HtmlBuilder exposing (HtmlBuilder, appendChild, appendChildren, prependAttribute, prependAttributes, prependChild, prependChildren)
```

Comment:
I'm not sure that this is the correct home for this module. Perhaps just `SemanticUI.HtmlBuilder`? (It's the module that used to be called `SemanticUI.Modules.Common`)